navigation/MainTabNavigator.js

```diff
@@ -79,10 +81,16 @@ const Home = createStackNavigator({
 })
 Home.navigationOptions = {
   tabBarIcon: ({ focused }) => (
-    <TabBarIcon focused={focused} name={'md-home'} />
+    <TabBarIcon focused={focused} name={'md-home'}/>
   ),
   tabBarOptions: {
-    activeTintColor: '#f18d1a'
+    activeTintColor: '#f18d1a',
```

Comment:
we should move the styling of navigations into styles.js or equivalent instead of doing 'isTablet' evaluation.